app/src/main/java/com/example/webservicewitharch/models/Pictures.java

```diff
@@ -38,4 +43,13 @@ public int getmLikes() {
         return mLikes;
     }
 
+    @NonNull
+    @Override
+    public String toString() {
```

Comment:
For Logging purposes overide the toString Method and get a clear representation of the object